### ISSUE TITLE
Question regarding output on /proc/self/fd

### ISSUE THREAD

**Issue by hojoonleokim** [SNU-ARC/2024_spring_sysprog_Lab2#32]
Hello
My code and reference code shows different result when input is directory "/proc/self/fd".
It seems like the process of closing fd is working differently between my code and the reference code. Is it acceptable to submit it as it is?
<img width="785" alt="Screenshot 2024-04-02 at 12 37 10 PM" src="https://github.com/SNU-ARC/2024_spring_sysprog_Lab2/assets/81799917/243516b1-27f9-464c-b5f9-98ab3a59022e">

**Comment by yunjayh** [comment]
Yes it can be different between the reference and your implementation. That's up to its implementation details.
It won't make any disadvantages in your lab grade.

**Comment by hojoonleokim** [comment]
Thank you for reply!
Have a good ones~
